shiny/http_server.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+
+
+app = Flask(__name__)
+
+
+@app.route('/v1/do-nothing/<unicorn_name>', methods=['POST'])
+def do_nothing(unicorn_name):
+    return ''
```

Comment:
It seems Pub/Sub returns an `Empty` on [some POST requests][1] so [I checked][2] what the JSON-over-HTTP interface returns and confirmed it is an empty dictionary.

[1]: https://github.com/googleapis/googleapis/blob/5cf7f5a30210bb5750d6918b30b288da2b2b91ec/google/pubsub/v1/pubsub.proto#L67-L80
[2]: https://gist.github.com/dhermes/3e94adb65ea0e11ba664b68e1b4d72d7